libsrc/telestrat/print.s

```diff
@@ -0,0 +1,16 @@
+; jede jede@oric.org 2017-01-22
+
+; void print (char * str);
+    .export         _print
+    .import         popax
+    .importzp       tmp1
+    .include        "telestrat.inc"
+
+.proc   _print
+    stx tmp1
+    ldy tmp1
+    BRK_TELEMON XWSTR0
```

Comment:
Yes, it's quickest way to write a string to video string. I removed this code, and i'll see in few day how to manage it